.eslintignore

```diff
@@ -0,0 +1,2 @@
+examples/sdk/node_modules/*
+examples/sdk/testpilot-metrics.js
```

Comment:
Ignoring this since it's a copied file (assuming the user does`npm run build` in the examples directory).